packages/web/src/utils/reward-utils.ts

```diff
@@ -37,3 +40,14 @@ export const isInternalRewardType = (rewardType: RewardType): boolean => {
 export const hasRewardInfo = <T>(data: Record<DisplayRewardType, T[]>): boolean => {
   return data.SWAP_FEE.length > 0 || data.INTERNAL_REWARD.length > 0 || data.EXTERNAL_REWARD.length > 0;
 };
+
+/**
+ * Functions to check reward claimability
+ * Returns true if the reward's claimable amount is greater than 0, otherwise false
+ * @param reward
+ * @returns boolean
+ */
+export const isClaimableReward = (reward: RewardModel): boolean => {
+  const claimableAmount = BigNumber(reward.claimableAmount || 0);
+  return !claimableAmount.isNaN() && claimableAmount.isGreaterThan(0);
+};
```

Comment:
When `BigNumber` has failed to convert a number, it returns `NaN`. Since `NaN` values will always return `false` when used with `isGreaterThan(0)`. So, there is no need to check for `NaN` separately.
```suggestion
export const isClaimableReward = (reward: RewardModel): boolean => {
  const claimableAmount = BigNumber(reward.claimableAmount ?? 0);
  return claimableAmount.isGreaterThan(0);
};
```